asdf.sh

```diff
@@ -7,7 +7,8 @@ else
 fi
 
 export ASDF_DIR
-ASDF_DIR="$(cd "$(dirname "$current_script_path")" &> /dev/null || exit 1; pwd)"
+ASDF_DIR="$(dirname "$current_script_path")"
+[ -d "$ASDF_DIR" ] || return
```

Comment:
The previous code executed `exit 1` if the path was not a directory. This code just executes `return`, which is not the same thing. I think this should be updated to be `[ -d "$ASDF_DIR" ] || exit 1`.

Reply:
Thanks for merging back upstream!

The reason for `return` is that, since asdf.sh is being sourced on .bashrc, if we `exit` we will crash user's session and they'll will never be able to spawn a shell.

Agree that `return` would fail silently and it would be better to explicit message user